### PR TITLE
https://wso2.org/jira/browse/BPS-1089

### PR DESCRIPTION
**Pull request by Subasinghe**

### CODE REVIEW

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/model/runtime/SignalEventReceivedRequest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2015. WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/model/runtime/SignalEventReceivedRequest.java]: Year should be 2016

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/SignalService.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2015. WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/SignalService.java]: Year 2016??

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/ProcessInstanceService.java [hunk 3]
```diff
@@ -597,6 +598,16 @@ public Response createOrUpdateExecutionVariable(@PathParam("processInstanceId")
         return result;
     }
 
+    @DELETE
```
Comment (vinok88) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/ProcessInstanceService.java]: Need Doc comments for all these API methods.

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/model/runtime/SignalEventReceivedRequest.java (hunk 1 above)
Comment (Subasinghe) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/model/runtime/SignalEventReceivedRequest.java]: Fixed in #474

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/SignalService.java (hunk 2 above)
Comment (Subasinghe) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/SignalService.java]: Fixed in #474

File: components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/ProcessInstanceService.java (hunk 3 above)
Comment (Subasinghe) [inline components/bpmn/org.wso2.carbon.bpmn.rest/src/main/java/org/wso2/carbon/bpmn/rest/service/runtime/ProcessInstanceService.java]: Fixed in #474

**Pull request merged**